docs/pages/server-access/guides/bpf-session-recording.mdx

```diff
@@ -163,8 +163,22 @@ ssh_service:
     # Optional: Controls where cgroupv2 hierarchy is mounted. Default value:
     # /cgroup2.
     cgroup_path: /cgroup2
+
+    # Optional: Controls the path inside cgroupv2 hierarchy where Teleport
+    # cgroups will be placed. Default value: /teleport
+    root_path: /teleport
 ```
 
+<Details title="Isolate session recordings">
+
+If you operate multiple Teleport instances on the same system, customize the
+`root_path` configuration to change the base cgroup slice path for session
+recordings resources. This ensures security isolation between recordings from
```

Comment:
I'm not sure what "session recordings resources" means. Should this be "session recordings"?

Reply:
I've updated it to match the title. [More info at my comment above.](https://github.com/gravitational/teleport/pull/38399#discussion_r1508195453)